libs/util.js

```diff
@@ -52,6 +52,17 @@ exports.authenticateAsAnonymous = function(req, res, next) {
   });
 };
 
+exports.forbidAnonymousUser = function (req, res, next) {
+  if (req.isAuthenticated()) {
+    const anonymousEmail = config.get('AnonymousEmail');
+    if (req.user.local.email !== anonymousEmail) {
+      return next();
+    }
+    req.logout();
```

Comment:
is logout necessary here?

Reply:
Yes, we need to logout anonymous user since redirecting to WATT page results in another redirect to project page once user is logged in. This breaks routing.